src/mono/wasm/debugger/BrowserDebugProxy/MemberReferenceResolver.cs

```diff
@@ -372,135 +372,152 @@ public async Task<JObject> Resolve(ElementAccessExpressionSyntax elementAccess,
                     rootObject = indexObject;
                     indexObject = null;
                 }
-                if (rootObject != null)
+                if (rootObject is null || (indexObject is null && elementAccess.ArgumentList is null))
+                    return null;
+
+                // 1. Parse the indexes
+                string elementIdxStr = null;
```

Comment:
For a follow up pr - this should be a `StringBuilder`.

Reply:
We can do it now because in the future PR we will need to change it anyway into some more complicated entity, so that we would be able to index with objects as well, not only strings.